### PR TITLE
docs(grid): visualize with debug

### PR DESCRIPTION
**Pull request by mtomcal**
## Description

After offline discussion we will document the `debug` toggle from Garden Grid in the `grid.stories.mdx` file. 

## Checklist

- [ ] :ok_hand: design updates are Garden Designer approved (add the
      designer as a reviewer)
- [x] :globe_with_meridians: demo is up-to-date (`yarn start`)
- [ ] :arrow_left: renders as expected with reversed (RTL) direction
- [ ] :metal: renders as expected with Bedrock CSS (`?bedrock`)
- [ ] :wheelchair: analyzed via [axe](https://www.deque.com/axe/) and evaluated using VoiceOver
- [ ] :guardsman: includes new unit tests
- [ ] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: packages/grid/demo/grid.stories.mdx [hunk 1]
```diff
@@ -18,6 +18,8 @@ import { GRID_ROWS as ROWS } from './stories/data';
 
 # Demo
 
+> Visualize the Grid component using the `debug` toggle in the Addons panel
```
Comment (hzhu) [inline packages/grid/demo/grid.stories.mdx:21]: Was this sentence meant to be a block quote?

**Review by hzhu — APPROVED** [review]
Nice I think this is helpful!

File: packages/grid/demo/grid.stories.mdx (hunk 1 above)
Comment (mtomcal) [reply packages/grid/demo/grid.stories.mdx:21]: I was thinking of using the block-quote as a "quick tip" style area.

Comment (jzempel) [inline packages/grid/demo/grid.stories.mdx:21]: [nits] the panel in the UI is named Controls. Ending sentence with a period.

**Review by jzempel — APPROVED** [review]

**Review by Francois-Esquire — APPROVED** [review]

**Pull request merged**